common/lib/xmodule/xmodule/course_module.py

```diff
@@ -834,6 +834,17 @@ class CourseFields(object):
         scope=Scope.settings,
     )
 
+    social_sharing_url = String(
```

Comment:
@stephensanchez and I discussed this out-of-band -- at this time there really isn't a better place to store this info -- maybe someday we'll see an actual Course entity in the ORM, which would really help to support these types of efforts.